tests/crm/test_crm.py

```diff
@@ -216,8 +215,9 @@ def verify_thresholds(duthost, **kwargs):
                 # For test case used 'nexthop_group' need to be configured at least 1 percent from available
                 continue
             used_percent = get_used_percent(kwargs["crm_used"], kwargs["crm_avail"])
-            if used_percent < 1:
-                logger.warning("CRM used entries is < 1 percent")
+            if used_percent < 1 or used_percent >= 100:
+                logger.warning("CRM used entries is < 1 or >= 100 percent")
```

Comment:
Can you add used_percent in the debug output?

Also, the wording is a bit confusing, is this intended use percentage or used percentage?

If this is intended use percentage, can we change the test to test a smaller number instead of skipping?

Reply:
Thanks. I've updated the log and added the used_percentage.
The test script is to configure the crm threshold type to percentage, and then insert a number of acl rules to trigger acm WARNING log. Therefore, the least used percentage is 1% to do the test. If we are limited to the available acl entry number, and the used percentage of acl group counter is below 1%, then the verification has to be skipped.